src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -1,5 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fields>
+        <fullName>InstallmentOppStageName__c</fullName>
+        <description>The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Opportunity Stage that NPSP applies to each new installment Opportunity when you create a Recurring Donation.</inlineHelpText>
+        <label>Stage Name for Installment Opportunities</label>
+        <length>50</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>IsEnhancementEnabled__c</fullName>
```

Comment:
@sam-knox I'm not so sure about this field name to define when RD2 is enabled. The field name begs the question "What Enhancement". How about `IsEnhancedRDEnabled__c`?

Reply:
The field is already in `npe03__Recurring_Donations_Settings__c` context so Label and Description give enough information, unless we are going to have another RD enhancement. However I have no objections to `IsEnhancedRDEnabled__c`.